### DISCUSSION TITLE
IPointerDown,IDragHandler, IPointerUpHandler 은 동작하지 않는건가요?

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#632] [5. Q&A]
해당 인터페이스를 implements하고, 각각 인터페이스를 구현했는데,

동작이 안되는 것 같습니다.

원래지원이 안되는 건지 다른설정이 필요한건지 알고 싶습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재는 지원하지 않습니다. 
지원할 수 있는 방법을 준비 중에 있습니다. 감사합니다. 🙏
